docs/src/demos/hooks/use-mouse/usage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useMantineTheme, Text, Code } from '@mantine/core';
+import { useMouse } from '@mantine/hooks';
+
+const code = `
+import { useMantineTheme, Text, Code } from '@mantine/core';
+import { useMouse } from '@mantine/hooks';
+
+function Demo() {
+  const theme = useMantineTheme();
+  const { x, y } = useMouse();
+
+  return (
+    <Text align="center" style={{ marginTop: theme.spacing.sm }}>
```

Comment:
Do not include irrelevant styles in demo code, remove `style={{ marginTop: theme.spacing.sm }}` from demo code and meaning won't change